$NetBSD: patch-CMakeLists.txt,v 1.3 2026/02/06 13:31:21 adam Exp $

- Don't set CMAKE_LIBRARY_PATH.
- Use policykit everywhere it makes sense.

--- CMakeLists.txt.orig	2026-02-01 10:38:53.000000000 +0000
+++ CMakeLists.txt
@@ -92,8 +92,6 @@ include(EttercapOSTest)
 #Check and see if we're running Darwin, specify the CMAKE_LIBARY_PATH to do so
 if(OS_DARWIN)
   set(CMAKE_SYSTEM_NAME Darwin)
-  set(CMAKE_LIBRARY_PATH ${SPECIAL_LIB_DIR}/lib ${CMAKE_LIBRARY_PATH})
-  set(CMAKE_INCLUDE_PATH ${SPECIAL_LIB_DIR}/include ${CMAKE_INCLUDE_PATH})
 endif()
 
 if(LIBRARY_BUILD)
@@ -112,7 +110,7 @@ set(INSTALL_LIBDIR ${CMAKE_INSTALL_FULL_
 set(INSTALL_DATADIR ${CMAKE_INSTALL_FULL_DATADIR} CACHE PATH "Data installation directory")
 set(INSTALL_EXECPREFIX ${CMAKE_INSTALL_PREFIX} CACHE PATH "")
 set(INSTALL_BINDIR  ${CMAKE_INSTALL_FULL_BINDIR} CACHE PATH "Binary files installation directory")
-if(OS_DARWIN OR OS_BSD_FREE OR OS_WINDOWS)
+if(NOT OS_LINUX)
   set(INSTALL_SYSCONFDIR ${CMAKE_INSTALL_PREFIX}/etc CACHE PATH "System configuration directory")
   set(POLKIT_DIR ${CMAKE_INSTALL_FULL_DATADIR}/polkit-1/actions/ CACHE PATH "Polkit installation directory")
 else()
